docs/reference/search/search.asciidoc

```diff
@@ -192,9 +192,29 @@ value cannot be `0`.
 (Optional, string) A comma-separated list of <field>:<direction> pairs.
 
 `_source`::
-(Optional, boolean)
-If `true`, the response includes the `_source` of matching documents under
-`hits`. Defaults to `true`.
+(Optional)
+Indicates which <<mapping-source-field,source fields>> are returned for matching
+documents. These fields are returned in the `hits._source` property of
+the search response. Defaults to `true`.
++
+.Valid values for `_source`
+[%collapsible%open]
+====
+`true`::
+(boolean)
+The entire document source is returned. This does not include:
```

Comment:
To me we should omit the part "This does not include ...", since this section is meant to be a simple API reference. That sentence is more of a 'note' or helpful reminder that the document source does not contain these values (which we already mention in the more detailed/ explanatory 'run a search' reference).